lib-es5/preloaded_file.js

```diff
@@ -68,7 +71,9 @@ var PreloadedFile = function () {
   }], [{
     key: "split_format",
     value: function split_format(identifier) {
-      var format, last_dot, public_id;
+      var format = void 0,
```

Comment:
Are we creating a new convention here or init variables with `void 0`?
If we do, maybe worth changing all around and add to lint?